login/src/main/resources/templates/web/change_password.html

```diff
@@ -4,11 +4,28 @@
       layout:decorator="layouts/main">
 <head>
     <th:block layout:include="nav :: head"></th:block>
+    <th:block layout:include="pw_validation :: head"></th:block>
 </head>
 <div class="header" layout:fragment="page-header">
     <th:block layout:include="nav :: nav"></th:block>
 </div>
 <div class="island" layout:fragment="page-content">
+  <script th:inline="javascript">
```

Comment:
I'm not sure it can be. Depending on when the template library injects `passwordPolicy`. But let me try.